spec/integration/transaction_spec.rb

```diff
@@ -3,7 +3,13 @@
     Dry.Transaction(container: container) do
       map :process
       step :verify
-      try :validate, catch: Test::NotValidError
+      try :validate, with: -> input {
+        if input[:email].nil?
+          raise(container[:invalid_error], "email required")
+        else
+          input
+        end
+      }, catch: Test::NotValidError
```

Comment:
I think `with:` isn't bad for a consistent feeling API for these step definitions, anyway. It's like "run this step _with_ users.operations.create_user" or "run this step _with_ [this proc I'm giving you right here]".

What do you think?